app/src/androidTest/java/org/mozilla/fenix/helpers/TestAssetHelper.kt

```diff
@@ -15,6 +15,7 @@ import org.mozilla.fenix.helpers.ext.toUri
 object TestAssetHelper {
     @Suppress("MagicNumber")
     val waitingTime: Long = TimeUnit.SECONDS.toMillis(15)
+    val pageLoadingTime = TimeUnit.SECONDS.toMillis(25)
```

Comment:
Is `waitingTime` not sufficient here? Do you think you'll re-use `pageLoadingTime` elsewhere?

Reply:
We had the need for it on Focus too, maybe it will be useful for verifying page content here as well. In this case, I saw that tabs weren't showing their title, I tried increasing the time for that.